ebuild-maintenance/removal/text.xml

```diff
@@ -113,16 +113,13 @@ In order to remove a virtual package, follow the following procedure:
     Wait the time appropriate for the last rites.
   </li>
   <li>
-    Update all ebuilds not to reference the virtual. Since there is
-    no urgent need to remove the virtual from user systems
-    and the resulting rebuilds would be unnecessary, do not bump ebuilds
-    when replacing the dependency.
+    Update all ebuilds not to reference the virtual with a new revision.
```

Comment:
This is just not very practical for dependencies that are via an eclass.

Also I think the argument about the compile time is still valid.

Reply:
It's no different for other non-virtual dependencies. A new revision is required. If you want to wait to do it until an ew version comes out of your package, that's fine, but you can't just elide the new revision. If you can suggest phrasing, go ahead.